### PR TITLE
Provider name links to source

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

**Review by lbiedinger — CHANGES_REQUESTED** [review]

File: plugins/europeana/record.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function extraFields(proxyData, edm, entities) {
   const providerAggregation = edm.aggregations[0];
   const europeanaAggregation = edm.europeanaAggregation;
   return lookupEntities(omitBy({
-    edmProvider: providerAggregation.edmProvider,
+    edmProvider: { link: providerAggregation.edmIsShownAt, value: providerAggregation.edmProvider },
```
[the remark this answers is not in the document]
Comment (alansutherland) [reply plugins/europeana/record.js:76]: ok thanks for your help

**Review by lbiedinger — APPROVED** [review]

File: components/item/MetadataField.vue [hunk 2]
```diff
@@ -33,13 +33,29 @@
             />
           </li>
         </template>
+        <template v-else-if="typeof fieldData.url !== 'undefined'">
```
Comment (mirjamverloop) [inline components/item/MetadataField.vue:36]: Would it work to just say `<template v-else-if="fieldData.url">`?

File: components/item/MetadataField.vue [hunk 3]
```diff
@@ -33,13 +33,29 @@
...
+          <li
+            :key="index"
+            :lang="langMappedValues.code"
+            data-qa="literal value"
```
Comment (mirjamverloop) [inline components/item/MetadataField.vue:40]: Is this correct? As in this case, it is not the literal value, right?

File: components/item/MetadataField.vue [hunk 4]
```diff
@@ -33,13 +33,29 @@
...
+          >
+            <SmartLink
+              :destination="fieldData.url"
+              :link-class="name === 'edmDataProvider' ? 'view-at' : null"
```
Comment (mirjamverloop) [inline components/item/MetadataField.vue:44]: Should this link show an icon as it is/could be an external link?

**Review by mirjamverloop — CHANGES_REQUESTED** [review]

File: components/item/MetadataField.vue (hunk 2 above)
Comment (alansutherland) [reply components/item/MetadataField.vue:36]: yes that does appear to suffice

File: components/item/MetadataField.vue (hunk 3 above)
Comment (alansutherland) [reply components/item/MetadataField.vue:40]: as discussed we'll change to 'entity link'

File: components/item/MetadataField.vue (hunk 4 above)
Comment (alansutherland) [reply components/item/MetadataField.vue:44]: ok I've added the icon

**Review by mirjamverloop — APPROVED** [review]

**Review by mirjamverloop — APPROVED** [review]

**Pull request merged**